src/sources/host_metrics/cpu.rs

```diff
@@ -40,6 +40,16 @@ impl HostMetrics {
                                         (String::from("cpu"), index.to_string()),
                                     ]),
                                 ),
+                                #[cfg(target_os = "linux")]
+                                self.counter(
+                                    name,
+                                    timestamp,
+                                    times.io_wait().get::<second>(),
+                                    BTreeMap::from([
+                                        (String::from("mode"), String::from("io_wait")),
```

Comment:
This new mode should get added to the documentation in `website/cue/reference/components/sources/host_metrics.cue`.

Reply:
Thanks for specifying this. Couldn't figure out the docs. 

Please let me know if there is anything else regarding this addition.